netbox/extras/views.py

```diff
@@ -1126,9 +1127,15 @@ def get(self, request, **kwargs):
         jobs_table = JobTable(
             data=script.jobs.all(),
             orderable=False,
-            user=request.user
+            user=request.user,
+            extra_columns=[(
+                'result', columns.TemplateColumn(
+                    template_code='<a href="{% url "extras:script_result" job_pk=record.pk %}">{{ record.completed }}</a>'
```

Comment:
We also need to explicitly pass the verbose name for each column now to support translation (e.g. `verbose_name=_('Result')`).